src/systems/ec/specs/components/footer/docs/usage.md

```diff
@@ -1,3 +1,22 @@
 ## Usage
 
-Do's and don'ts
+The footer is located at the bootom of a page. It provides inforamtion from the European Commission. 
+
+### Main content area
+
+The main content area of the footer offers access to the political and the informational section of the European Commission. It also contain links to the European Union and the EU institutions.
+Users can also find information of copyright, the Commission's new web presence, resources for partners, cookies, legal notices, contact and social media.
+
+### Custom area
+
+In some circumstances, a custom are can be added if the main and secondary content areas do not satisfy the requirements.  A custom area can contain:
+
+* idntification of the site name
+* iformation on how to connect to social platforms
```

Comment:
I also realize that when I try to edit the sentence then the text afterward begin to disappear. This is why I have missing letters of the words...need to still get used to it. (That wasn't so smart in my view, the way it worked, or I just haven't figured out how to do it.)